data/structures.yml

```diff
@@ -95,7 +95,7 @@ BE:
   :national_id_length: 3
   :bban_format: "\\d{3}\\d{7}\\d{2}"
```

Comment:
I think that if the account number is now 7 - 2 - 3, and this is bank code + account number, this should be 3 - 7 - 2 **- 3**. But not totally clear on what this is used for tbh.

Reply:
Think this is only used to validate the IBANs not extract the various parts this is how we're currently extracting the parts from the IBAN. https://github.com/gocardless/ibandit/pull/179/commits/d173a5905efb57b9b6c9922d742511839693f2b3 shows how we're currently handling these